k8s/agones/scripts/agones-installer.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
Can you reminder me why you didn't patch [`/bin/deploy.sh`](https://github.com/GoogleCloudPlatform/click-to-deploy/blob/c213b06eab2798b6db407d5c309bd2147d307284/k8s/wordpress/deployer/Dockerfile#L20-L23)? Is is because deployer doesn't have enough permissions to create a CRD?

Reply:
The account used by deploy.sh has cluster-admin only in it's own namespace. That prevents, among other things, installing the CRDs and webhooks.

Reply:
Have to tried to patch the deployer that apply installer.yaml and continue it work?

This way we would not have to have to scale the application down and so on.

Reply:
I can't do that without major changes to the Helm chart in Agones.

The chart rendering generates a set of CAs with common names that get inserted into the certificate that is stored inside of a secret that the controller uses. That certificate is used by the webhook, and that cannot be installed by the original deployer since it doesn't have cluster wide permissions. The only workaround so far is to do what I'm doing above, until the marketplace can better support CRDs and webhooks.